### PR TITLE
Npm publish

### PR DESCRIPTION
**Pull request by Mxchaeltrxn**
Addresses #100. 

I have done everything in #100 except fixing the build badge—there [doesn't seem to be an easy way to get this working](https://github.com/vercel/vercel/discussions/4501#discussioncomment-19401) so I just wanted to confirm if this is the approach you wanted to take. I'm asking because I've never done it this way before and it seems somewhat involved.

As for the rest of the tasks, everything went well though for the npm publish, I thought I didn't publish v0.1.0-0 correctly so I deleted it and republished. However, I [could not republish it as v0.1.0-0](https://github.com/npm/npm/issues/9266) so I published it as 0.2.0-0 (in retrospect, maybe it should have been 0.1.0-1). I don't think it's a huge deal since it's dev but it's just slightly ugly/strange, I guess.

Also, I ran `npm run release --tag=dev` because `yarn release --tag=dev` did not work for me.

I was going to post an image of the folder structure after installing it as a package but you've already tried!

### CODE REVIEW

**Comment by haydn** [comment]
@Mxchaeltrxn Sorry, I should have been clearer. The badge is the TravisCI badge that appears in the README:

image

Says "build unknown" at the moment cause it's not pointing to the updated repo (https://travis-ci.org/github/sketchbook-js/sketchbook).

File: package.json [hunk 1]
```diff
@@ -1,8 +1,21 @@
 {
-  "name": "sketchbook-js",
-  "version": "0.1.0",
-  "private": true,
-  "bin" : { "sketchbook" : "cli/index.js" },
+  "name": "@sketchbook-js/sketchbook",
+  "version": "0.2.0-0",
+  "description": "The design tool that lives in your repo.",
+  "repository": "https://github.com/haydn/sketchbook-js.git",
```
Comment (haydn) [inline package.json:5]: This is still referencing the old repo.

File: package.json [hunk 2]
```diff
@@ -1,8 +1,21 @@
...
+  "keywords": [
+    "design",
+    "tool",
+    "graphic"
+  ],
+  "main": "cli/index.js",
+  "author": "Hayden Ewers",
+  "license": "MIT",
+  "bugs": "https://github.com/haydn/sketchbook-js/issues",
+  "homepage": "https://github.com/haydn/sketchbook-js#readme",
```
Comment (haydn) [inline package.json:15]: Also got the old repo here

**Comment by Mxchaeltrxn** [comment]
Sorry that was clumsy of me. I've fixed it with a few pushes. Sorry I wasted some build minutes. I don't think I can cancel the builds because I don't have access.  

Anyway, I'm not sure why the build still shows up as cancelled. It worked when I used the [travis url](https://api.travis-ci.org/sketchbook-js/sketchbook.svg?branch=master) but I used shields simply because you did.

**Comment by haydn** [comment]
@Mxchaeltrxn I think that main Travis CI check wasn't passing because the branch wasn't up-to-date with master. Just updated it now.

**Comment by haydn** [comment]
@Mxchaeltrxn Well that wasn't the reason, but I've removed that extra Travis CI check from the [branch protection rules](https://github.com/sketchbook-js/sketchbook/settings/branches) and made the `pr` and `push` checks required instead. I'm guessing Travis CI changed how the integration works at some point and moving across to a new repo has surfaced it. 🤷

**Comment by Mxchaeltrxn** [comment]
@haydn I've done the last checkbox in issue #100!

**Comment by haydn** [comment]
@Mxchaeltrxn Merged!

I tested publishing another version on master after merging, great success:

https://unpkg.com/@sketchbook-js/sketchbook@0.2.0-1/

You probably encountered this too, there's an issue with `np` at the moment that incorrectly warns about new files being added:

https://github.com/sindresorhus/np/issues/577

**Pull request merged**